libs/components/pages/testing/src/modules/link-list/link-list-harness.spec.ts

```diff
@@ -87,4 +89,35 @@ describe('Link list harness', () => {
     fixture.componentRef.setInput('showLinks', false);
     await expectAsync(harness.isVisible()).toBeResolvedTo(false);
   });
+
+  it('should retrieve list items', async () => {
+    const { fixture, loader } = await setupTest();
+    fixture.componentRef.setInput('inputLinks', [
+      {
+        label: 'Test Link 1',
+        permalink: {
+          url: '#',
+        },
+      },
+      {
+        label: 'Test Link 2',
+        permalink: {
+          url: '#',
+        },
+      },
+    ]);
+    fixture.detectChanges();
+
+    const links = await loader.getAllHarnesses(SkyLinkListItemHarness);
```

Comment:
Since teechnically consumers can put `sky-link-list-item` anywhere, can we add a helper function for SkyLinkListHarness and SkyNeedsAttentionHarness that gets all item harness instances inside action hub only.

Reply:
Looking at the [documentation for `ComponentHarness.locatorFor`](https://material.angular.io/cdk/testing/api#ComponentHarness), it should already handle scoping to the current harness, right?

This test is not scoping and asserts the total number of link items in the test component.

Reply:
ooooh neat! TIL that so thatll simplify harnesses in the future, neat!